src/test/resources/stub/mappings/bizEventsService/bizEvents_Get_TransactionsList_OK.json

```diff
@@ -0,0 +1,42 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPathPattern": "/bizEventsMock/transactions",
+    "headers": {
+      "x-api-key": {
+        "matches": "x_api_key([0-9]?)+"
+      },
+      "x-fiscal-code": {
+        "matches": "DUMMY_FISCAL_CODE"
+      },
+      "x-continuation-token": {
+        "matches": "TOKEN"
+      }
+    },
+    "queryParameters": {
+      "size": {
+        "equalTo": "1"
+      }
+    }
+  },
+  "response": {
+    "status": 200,
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "jsonBody": {
+      "transactions": [
+        {
+          "transactionId": "1",
+          "payeeName": "Comune di Milano",
+          "payeeTaxCode": "MI_XXX",
+          "amount": "180,00",
+          "transactionDate": "2024-03-27T13:07:25Z",
+          "isCart": false,
+          "isPayer": true,
+          "isDebtor": true
+        }
+      ]
+    }
```

Comment:
i remember bizEvents returns an array (not an object with a array property), am i wrong?

Reply:
yes that's wrong, the biz-events api returns an object with the transactions property inside which is an array of transactions.